opa-manager-operator/actions.yaml

```diff
@@ -0,0 +1,3 @@
+reload-config:
```

Comment:
I can't find an implementation for this action. Where is it?

Reply:
This action still appears to not exist. Please remove it from actions.yaml if there is no implementation.

Reply:
removed file